.github/workflows/release-templates.yaml

```diff
@@ -0,0 +1,339 @@
+        uses: cargo-generate/cargo-generate-action@v0.17.5
+        with:
+          name: ${{ env.PROJECT_NAME }}
+          template: ${{ matrix.crates }}
+          other: "-d description=Example-credential-registry -d template_type=full"
+
+      - name: Install toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ env.RUST_VERSION }}
+
+      # Run diff to check if there are any differences between the generated smart contract
+      # and the example smart contract. The template have hardcoded versions of `concordium-std`
+      # which needs to be replaced with the path used in the smart contract example folder.
+      # This CI pipeline uses `root` as the author which needs to be replaced with the author
+      # `Concordium <developers@concordium.com>` of the smart contract examples.
+      - name: Run diff
+        run: |
+          mv $PROJECT_NAME ${{ runner.temp }}/
+          sed -i "s/root/Concordium <developers@concordium.com>/g" ${{ runner.temp }}/$PROJECT_NAME/Cargo.toml
+          sed -i "s/version = \"10.0\", default-features = false/path = \"..\/..\/concordium-std\", version = \"10.0\", default-features = false/g" ${{ runner.temp }}/$PROJECT_NAME/Cargo.toml
+          sed -i "s/version = \"6.1\", default-features = false/path = \"..\/..\/concordium-cis2\", version = \"6.1\", default-features = false/g" ${{ runner.temp }}/$PROJECT_NAME/Cargo.toml
+          sed -i "s/concordium-smart-contract-testing = \"4.2\"/concordium-smart-contract-testing = {path = \"..\/..\/contract-testing\"}/g" ${{ runner.temp }}/$PROJECT_NAME/Cargo.toml
+          sed -i "s/concordium-std-derive = \"6.0\"/concordium-std-derive = {path = \"..\/..\/concordium-std-derive\"}/g" ${{ runner.temp }}/$PROJECT_NAME/Cargo.toml
+          diff ${{ runner.temp }}/$PROJECT_NAME/Cargo.toml examples/credential-registry/Cargo.toml
+          diff ${{ runner.temp }}/$PROJECT_NAME/src/lib.rs examples/credential-registry/src/lib.rs
+          diff ${{ runner.temp }}/$PROJECT_NAME/tests/tests.rs examples/credential-registry/tests/tests.rs
+          diff ${{ runner.temp }}/$PROJECT_NAME/README.md examples/credential-registry/README.md
+
+  release:
+    name: Releases
+    runs-on: ubuntu-latest
+    needs: 
+      - cargo-generate-templates
+      - cargo-generate-credential-registry-template
+      - clippy-template
+      - clippy-credential-registry-template
+      - cargo-generate-cis2-nft-template-comparison
+      - cargo-generate-credential-registry-template-comparison
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Create a GitHub release
+        uses: ncipollo/release-action@v1
+        with:
+          tag: releases/templates/v${{ github.event.inputs.tag }}
+          draft: true
+          name: Template Release v${{ github.event.inputs.tag }}
+          body: ${{ steps.tag_version.outputs.changelog }}
```

Comment:
I don't think this works, `steps.tag_version` is referring to a step with `id: tag_version`, which is not here.

Reply:
Yes, this is a mistake, the body was meant to be empty. Otherwise, we should parse the changelog and get changes from there, but that would be a little much for this.